packages/react-widgets/src/widgets/TableWidget.js

```diff
@@ -121,6 +122,7 @@ function TableWidget({
             height={height}
             dense={dense}
             isLoading={isLoading}
+            onRowClick={onRowClick}
```

Comment:
I've added support for the `onRowClick` event Handler to the TableWidget, because it was support but unused, and it's very useful for testing this on the `cloud-native` side 👍

Reply:
ok but add a new comment in the parm section please